### ISSUE TITLE
run_tf_glue with AdamW optimizer and distributed training

### ISSUE THREAD

**Issue by BOUALILILila** [huggingface/transformers#3030]
# ❓ Questions & Help
Can you please update the run_tf_glue example (Tensorflow version) that includes the Adam optimizer with Warm-up and weight decay.  If you can also show how to run the code on multiple GPUs and on TPU.
